_includes/download.html

```diff
@@ -27,8 +27,8 @@
                         <h3>Workshops</h3>
                         <ul>
                           <li>Foam Roller Demos</li>
-                          <li>Yoga fundamentals</li>
-                          <li>Stretching</li>
+                          <li>Yoga Fundamentals</li>
+                          <li>Mindful Stretching</li>
```

Comment:
This is not able to be merged in because `Mindful Stretching` is already in.

<img width="336" alt="screen shot 2016-03-17 at 1 55 02 pm" src="https://cloud.githubusercontent.com/assets/96776/13860913/e82b1c6e-ec47-11e5-891b-996ae861c053.png">

I think the only change required here would be to just change `fundamentals` to `Fundamentals`.

You may want to submit a new PR with that.